examples/flax/language-modeling/run_clm_flax.py

```diff
@@ -558,8 +582,11 @@ def eval_step(params, batch):
     logger.info(f"  Num examples = {len(train_dataset)}")
     logger.info(f"  Num Epochs = {num_epochs}")
     logger.info(f"  Instantaneous batch size per device = {training_args.per_device_train_batch_size}")
-    logger.info(f"  Total train batch size (w. parallel & distributed) = {train_batch_size}")
-    logger.info(f"  Total optimization steps = {total_train_steps}")
+    logger.info(
+        f"  Total train batch size (w. parallel & distributed) = {train_batch_size * training_args.gradient_accumulation_steps}"
+    )
+    logger.info(f"  Total global steps = {total_steps}")
+    logger.info(f"  Total optimization steps = {total_optimization_steps}")
```

Comment:
We should probably also update

```cur_step = epoch * (len(train_dataset) // train_batch_size) + step``` in the line below to use `total_batch_size` no? 
Think when someone uses `gradient_accumulation` parameters like `logging_steps` should have have a different meaning as well no?

Reply:
Yes logging should be done every `logging_steps` full training steps, same for saves if you use `save_steps` in those scripts.